package.json

```diff
@@ -62,22 +62,39 @@
 		"vscode:prepublish": "npm run lint && npm run build",
 		"build": "tsc --project server/tsconfig.json && tsc --project client/tsconfig.json",
 		"lint:ts": "tsc --noEmit --project server/tsconfig.json && tsc --noEmit --project client/tsconfig.json && eslint --cache .",
-		"lint:json": "v8r -s server/src/data/.schema.json server/src/data/*.json",
-		"lint": "npm run lint:ts && npm run lint:json"
+		"lint:json": "v8r -s server/src/data/schema/info.json server/src/data/*.json",
+		"lint": "npm run lint:ts && npm run lint:json",
+		"package": "vsce package"
 	},
 	"dependencies": {
 		"@bhsd/common": "^0.4.4",
+		"@types/node": "^22.9.0",
 		"color-rgba": "^3.0.0",
 		"vscode-languageclient": "^9.0.1",
 		"vscode-languageserver": "^9.0.1",
 		"vscode-languageserver-textdocument": "^1.0.12",
 		"wikilint": "^2.13.4"
 	},
 	"devDependencies": {
+		"@stylistic/eslint-plugin": "^2.10.1",
 		"@types/color-rgba": "^2.1.3",
 		"@types/vscode": "^1.95.0",
+		"@typescript-eslint/eslint-plugin": "^8.14.0",
```

Comment:
Version 8 is incompatible with my ESLint configurations. Please downgrade it to version 7.